nim/templates/index.html

```diff
@@ -29,27 +29,43 @@ <h1>So it begins...</h1>
 
     <div id=piles></div>
 
-
     <div style="clear: both;">
       <button onclick="updateGameState()">Make Move</button>
+    </div>
+
+    <div>
+      <input type="number" id="pile" value="1" min="1">
+      pile
+    <div>
+    </div>
+      <input type="number" id="stones" value="1" min="1">
+      stones
+    </div>
```

Comment:
Looks like those should be before the `Make Move` button. Also same thing for below, the inputs should be before the `New Game` button.

Reply:
Done! :)